spec/models/miq_group_spec.rb

```diff
@@ -270,14 +270,13 @@
     it "adds new groups after initial seed" do
       [Tenant, MiqUserRole, MiqGroup].each(&:seed)
 
-      role_map_path = File.expand_path(File.join(Rails.root, "db/fixtures/role_map.yaml"))
+      role_map_path = Rails.root.join("db/fixtures/role_map.yaml")
```

Comment:
I was avoiding the dependency of using `ApplicationRecord` in the spec, but I suppose it doesn't matter...sure :+1: